src/common/Header/index.js

```diff
@@ -14,18 +14,27 @@ import {
   SearchInput,
   ListLink,
 } from "./styled";
-import { useReplaceQueryParameter } from "../../queryParameters";
+import { useQueryParameter, useReplaceQueryParameter } from "../../queryParameters";
 import { useLocation } from "react-router";
 
 export const Header = () => {
   const location = useLocation();
   const replaceQueryParameter = useReplaceQueryParameter();
+  const pageParameter = +useQueryParameter("page");
+  const page = pageParameter > 1 || pageParameter === 0 ? 1 : pageParameter;
 
   const onInputchange = debounce(({ target }) => {
     replaceQueryParameter({
       value: target.value.trim(),
       key: "search"
-    })
+    });
+
+    replaceQueryParameter({
+      value: page,
```

Comment:
I think you can just type `value: 1,` And then you don't need `pageParameter` and `page` 😀

Reply:
You are right, thank you